storage/replica_command.go

```diff
@@ -1322,7 +1348,7 @@ func (r *Replica) AdminMerge(args proto.AdminMergeRequest, desc *proto.RangeDesc
 		})
 		return txn.Run(b)
 	}); err != nil {
-		return reply, util.Errorf("merge of range %d into %d failed: %s", subsumedDesc.RangeID, desc.RangeID, err)
+		return reply, util.Errorf("merge of range into %d failed: %s", desc.RangeID, err)
```

Comment:
there's a lot of `util.Errorf` going on here. when these errors emerge, do they make any sense?